client/src/AppBundle/Resources/views/Macros/macros.html.twig

```diff
@@ -318,3 +318,50 @@
         </div>
     </details>
 {% endmacro %}
+
+{% macro successBanner(successText, transDomain, successHeading) %}
+    <div class="moj-banner moj-banner--success">
+        <div class="moj-banner__message">
+            {% if successHeading is not null %}
+                <h3 class="govuk-heading-m">
+                    <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+                        <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z" />
+                    </svg>
```

Comment:
Would be nice to put these icons into the `icon()` macro above, so they can be reused more easily.
```suggestion
                   {{ _self.icon('success', 'moj-banner__icon') }}
```

Reply:
Agree - I'll break them out.